virt-v2v/cold/entrypoint.go

```diff
@@ -220,39 +205,48 @@ func executeVirtV2v(source string, args []string) (err error) {
 	}
 
 	virtV2vMonitorCmd := exec.Command("/usr/local/bin/virt-v2v-monitor")
-	virtV2vMonitorCmd.Stdin = teeOut
+	virtV2vMonitorCmd.Stdin = r
 	virtV2vMonitorCmd.Stdout = os.Stdout
-	virtV2vMonitorCmd.Stderr = os.Stderr
 
 	if err = virtV2vMonitorCmd.Start(); err != nil {
 		fmt.Printf("Error executing monitor command: %v\n", err)
 		return
 	}
 
-	if source == OVA {
-		scanner := bufio.NewScanner(teeErr)
-		const maxCapacity = 1024 * 1024
-		buf := make([]byte, 0, 64*1024)
-		scanner.Buffer(buf, maxCapacity)
-
-		for scanner.Scan() {
-			line := scanner.Bytes()
-			if match := UEFI_RE.FindSubmatch(line); match != nil {
-				fmt.Println("UEFI firmware detected")
-				firmware = "efi"
+	done := make(chan error, 1)
+	go func() {
+		if source == OVA {
+			scanner := bufio.NewScanner(r)
+			const maxCapacity = 1024 * 1024
+			buf := make([]byte, 0, 64*1024)
+			scanner.Buffer(buf, maxCapacity)
+
+			for scanner.Scan() {
+				line := scanner.Bytes()
+				if match := UEFI_RE.FindSubmatch(line); match != nil {
+					fmt.Println("UEFI firmware detected")
+					firmware = "efi"
+				}
 			}
-		}
 
-		if err = scanner.Err(); err != nil {
-			fmt.Println("Output query failed:", err)
-			return err
+			if err := scanner.Err(); err != nil {
+				fmt.Println("Output query failed:", err)
+				done <- err
+				return
+			}
 		}
-	}
+		done <- nil
+	}()
```

Comment:
Its only a workaround until virt-v2v will relese a fix (not included in main branch), since the server that sets the vm configuration is set up here it made scene to have this scan in this file as well but I did thought of that initially.